components/demo/code-view.js

```diff
@@ -36,7 +36,9 @@ class CodeView extends LitElement {
 			const path = `/node_modules/prismjs/components/prism-${language}.min.js`;
 			this._dependenciesPromise = import(path);
 		}
-		this._updateCode(this.shadowRoot.querySelector('slot'));
+		if (this.shadowRoot) {
+			this._updateCode(this.shadowRoot.querySelector('slot'));
+		}
```

Comment:
This was unrelated to the animate stuff, but noticed a JS error on the animate demo page -- it's accessing `shadowRoot` before things have rendered.

I think it'd be safer to do this work in `updated()` anyway, so might be worth looking into that.